src/Raven.Client/Documents/Commands/NextIdentityForCommand.cs

```diff
@@ -1,47 +0,0 @@
-﻿using System;
-using System.Net.Http;
-using Raven.Client.Http;
-using Raven.Client.Util;
-using Sparrow.Json;
-
-namespace Raven.Client.Documents.Commands
```

Comment:
We cannot remove those. Need to stay in the same place as they were.

Reply:
done